discovery-provider/integration_tests/tasks/test_index_tastemaker_notification.py

```diff
@@ -0,0 +1,207 @@
+from datetime import datetime, timedelta
+
+from integration_tests.utils import populate_mock_db
+from sqlalchemy import asc
+from src.models.notifications.notification import Notification
+from src.tasks.index_tastemaker_notifications import index_tastemaker_notifications
+from src.utils.config import shared_config
+from src.utils.db_session import get_db
+
+REDIS_URL = shared_config["redis"]["url"]
+
+BASE_TIME = datetime(2023, 1, 1, 0, 0)
+
+
+def test_index_tastemaker_notification_no_trending(app):
+    with app.app_context():
+        db = get_db()
+
+        # Add some users to the db so we have blocks
+        entities = {
+            "tracks": [{"track_id": i, "owner_id": 3} for i in range(5)],
+        }
+        populate_mock_db(db, entities)
+
+        index_tastemaker_notifications(
+            db=db,
+            top_trending_tracks=[],
+            tastemaker_notification_threshold=2,
+        )
+        with db.scoped_session() as session:
+            notifications = (
+                session.query(Notification)
+                .filter(Notification.type == "tastemaker")
+                .order_by(asc(Notification.specifier))
+                .all()
+            )
+            assert len(notifications) == 0
+
+
+def test_index_tastemaker_notification_no_tastemakers(app):
+    with app.app_context():
+        db = get_db()
+
+        # Add some users to the db so we have blocks
+        entities = {
```

Comment:
just wondering if we need this much data for a case that's checking if no index happens, we dont generate?

Reply:
u righttt